jsonrpc/src/common/messages.ts

```diff
@@ -20,7 +20,7 @@ export interface RequestMessage extends Message {
 	/**
 	 * The request id.
 	 */
-	id: number | string;
+	id: number | string | null;
```

Comment:
It was more for me to avoid that I ever create a request with a `id` being `null`. Since the library doesn't allow sending requests directly it will not cause harm. But I with change the code for `createRequestQueueKey` and throw if I receive a message with id null.